packages/PreviewCard/PreviewCard.jsx

```diff
@@ -0,0 +1,160 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+
+import Box from '@tds/core-box'
+import Text from '@tds/core-text'
+import { colorTelusPurple, colorWhiteLilac } from '@tds/core-colours'
+import safeRest from '@tds/shared-safe-rest'
+
+import warn from '../../shared/utils/warn'
+
+const Anchor = styled.a`
+  text-decoration: none;
+  cursor: pointer;
+`
+
+const BoxContainer = styled.div`
+  background-color: white;
+  width: 100%;
+  opacity: 1;
+  border: 1px solid ${colorWhiteLilac};
+  border-radius: 4px;
+  height: ${props => (props.header || props.footer ? '465px' : '350px')};
+  @media (max-width: 576px) {
+    height: auto;
+  }
+  overflow: hidden;
+  &:hover {
+    box-shadow: 0 0 16px 0 rgba(213, 213, 213, 0.5);
+  }
+  display: inline-block;
+`
+
+const ContentContainer = styled.div`
+  line-height: ${props => (props.header && props.footer ? '21px' : undefined)};
+`
+
+const ImageContainer = styled.div`
+  border-radius: 4px 4px 0 0;
+  overflow: hidden;
+  line-height: 0px;
+  max-height: 250px;
+  @media (max-width: 992px) {
+    height: auto;
+  }
+  img {
+    transition: all 0.3s ease-in-out !important;
+  }
+  &:hover img {
+    transform: scale(1.03);
+  }
+`
+
+const P = styled.p`
+  color: ${colorTelusPurple};
+`
+
+/**
+ * The PreviewCard component creates the appearance of a page snippet, and can be used in a list format.
+ * @version ./package.json
+ */
+const PreviewCard = ({ category, other, image, body, footer, href, linkComponent, ...rest }) => {
+  if (rest.to && !(linkComponent && rest.to)) {
+    warn('Link', 'The props `linkComponent` and `to` must be used together.')
+  }
+
+  let newBody = body
+  if (body.length > 70) {
+    newBody = `${body.substr(0, 70)}...`
+  }
+
+  let header = category || other
+  if (category && other) {
+    header = `${category} \u00B7 ${other}`
+  }
+
+  const innerCard = (
+    <BoxContainer header={header} footer={footer}>
```

Comment:
This is purely programming preference, but it would be clearer if we pass _taller_ prop instead of _header_ & _footer_ prop to `<BoxContainer />`. This way `<BoxContainer />` is a pure presentational component which has no knowledge of parent props such as _header_ and _footer_.

```
const hasHeaderOfFooter = header || footer
<BoxContainer taller={hasHeaderOrFooter} />
<ContentContainer taller={hasHeaderOrFooter} />
```

Reply:
Agreed